src/ui/exile_modal.py

```diff
@@ -0,0 +1,47 @@
+import discord
+from services.exile_service import exile_user
+from util import calculate_time_delta
+from .helper import create_modal_embed
+
+
+class ExileModal(discord.ui.Modal):
+    def __init__(self, user: discord.Member) -> None:
+        super().__init__(title=f"Exile User {user.display_name}")
+        self.user = user
+
+    duration = discord.ui.TextInput(label="Exile Duration")
+
+    reason = discord.ui.TextInput(
+        label="Exile Reason",
+        style=discord.TextStyle.long,
+        placeholder="Why is the user exiled",
```

Comment:
same thing here, "Reason for exiling user"

Reply:
Done